AT/simulator_thread/scenario_setting_user_script.py

```diff
@@ -0,0 +1,19 @@
+from AT.simulator_thread.scenario_tools import add_variable
```

Comment:
this should probably be reworked to json files that do not require restarting the server for changes

Reply:
I was planning to ask you regarding why using json files when you can simply use python files. I guess this is the answer. This is not happening soon though.